DEV-INTRO.md

```diff
@@ -55,7 +62,11 @@ The steps in this section only need to be done the fist time you start working w
 
 #### .env file
 
-You can copy the [`env.example`](docker/env.example) file as .env in your pecan folder. The variables we want to modify are:
+You can copy the [`docker/env.example`](docker/env.example) file as .env in your pecan folder. The variables we want to modify are:
+
+```sh
```

Comment:
make sure to add the windows version as well

Reply:
added

Reply:
done